test/unit/mdc-dialog/foundation.test.js

```diff
@@ -382,7 +381,7 @@ test('interaction does not close dialog with action for non-activation keys', ()
   td.when(mockAdapter.getActionFromEvent(event)).thenReturn(action);
 
   foundation.open();
-  foundation.handleInteraction(event);
+  foundation.handleKeydown(event);
```

Comment:
Using `Escape` key for this test case seems consuming since Esc key is handled separately in `handleDocumentKeydown`. Can we use a different key?